vendor/egl/egl.odin

```diff
@@ -0,0 +1,61 @@
+//+build linux
+package egl
+
+NativeDisplayType :: rawptr
+NativeWindowType  :: rawptr
+Display :: rawptr
+Surface :: rawptr
+Config  :: rawptr
+Context :: rawptr
```

Comment:
Do these need to be `distinct` or not?

Reply:
Probably should. I will sort this out this weekend